src/helpers.ts

```diff
@@ -0,0 +1,27 @@
+import { Todo } from './types/Todo';
+import { FilterBy } from './types/FIlterBy';
+
+export const getVisibleTodos = (
+  todos:Todo[],
+  value: string,
+  filter: FilterBy,
+) => {
+  let copy = [...todos];
+
+  switch (filter) {
+    case FilterBy.Active:
+      copy = todos.filter(todo => !todo.completed);
+      break;
+
+    case FilterBy.Completed:
+      copy = todos.filter(todo => todo.completed);
+      break;
+
+    default:
+      break;
+  }
```

Comment:
I believe there's a way to rewrite it using only one filter method and ternary operator/if block